### PR TITLE
en,zh: Adjust for v1 CRD

### PR DESCRIPTION
**Pull request by KanShiori**
### What is changed, added, or deleted? (Required)

Merge after PR https://github.com/pingcap/tidb-operator/pull/4151

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

Fix https://github.com/pingcap/docs-tidb-operator/issues/1464
- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: zh/get-started.md [hunk 1]
```diff
@@ -262,6 +262,10 @@ customresourcedefinition.apiextensions.k8s.io/tidbinitializers.pingcap.com creat
 customresourcedefinition.apiextensions.k8s.io/tidbclusterautoscalers.pingcap.com created
 ```
 
+> **注意：**
+> 
+> Kubernetes 1.16 之前版本，仅支持 v1beta1 版本的 CRD，需要上述命令中文件 "crd.yaml" 改为 "crd_v1beta1.yaml"。l
```
Comment (DanielZhangQD) [inline zh/get-started.md:267]:
```suggestion
> Kubernetes 1.16 之前版本，仅支持 v1beta1 版本的 CRD，需要上述命令中文件 "crd.yaml" 改为 "crd_v1beta1.yaml"。
```

File: zh/upgrade-tidb-operator.md [hunk 2]
```diff
@@ -17,12 +17,23 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/upgrade-tidb-operator/']
...
+    * 如果 Kubernetes 版本大于等于 1.16:
 
-    ```shell
-    kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/v1.2.4/manifests/crd.yaml && \
-    kubectl get crd tidbclusters.pingcap.com
-    ```
+         {{< copyable "shell-regular" >}}
+
+         ```shell
+         kubectl replace -f https://raw.githubusercontent.com/pingcap/tidb-operator/v1.2.4/manifests/crd.yaml && \
```
Comment (DanielZhangQD) [inline zh/upgrade-tidb-operator.md:25]: The changes will not be in v1.2.

File: zh/upgrade-tidb-operator.md [hunk 3]
```diff
@@ -17,12 +17,23 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/upgrade-tidb-operator/']
...
+         kubectl get crd tidbclusters.pingcap.com
+         ```
+    
+     * 如果 Kubernetes 版本小于 1.16:
+
+         {{< copyable "shell-regular" >}}
+
+         ```shell
+         kubectl replace -f https://raw.githubusercontent.com/pingcap/tidb-operator/v1.2.4/manifests/crd_v1beta1.yaml && \
```
Comment (DanielZhangQD) [inline zh/upgrade-tidb-operator.md:34]: Is this case verified?

File: zh/upgrade-tidb-operator.md [hunk 4]
```diff
@@ -103,7 +124,7 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/upgrade-tidb-operator/']
         {{< copyable "shell-regular" >}}
 
         ```shell
-        kubectl apply -f ./crd.yaml
+        kubectl replace -f ./crd.yaml
```
Comment (DanielZhangQD) [inline zh/upgrade-tidb-operator.md:127]: Should we rename the file in L97?

Comment (KanShiori) [reply zh/upgrade-tidb-operator.md:127]: then flag `wget -O` rename the file

File: zh/upgrade-tidb-operator.md (hunk 3 above)
Comment (KanShiori) [reply zh/upgrade-tidb-operator.md:34]: not yet, I will have a try

File: zh/upgrade-tidb-operator.md (hunk 2 above)
Comment (KanShiori) [reply zh/upgrade-tidb-operator.md:25]: the version in URL will be replaced after release v1.3

File: zh/deploy-tidb-from-kubernetes-gke.md [hunk 5]
```diff
@@ -94,10 +94,14 @@ kubectl get nodes
...
 
 ```shell
-kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/crd.yaml && \
+kubectl create -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/crd.yaml && \
 kubectl get crd tidbclusters.pingcap.com
 ```
 
+> **注意：**
+> 
+> Kubernetes 1.16 之前版本，仅支持 v1beta1 版本的 CRD，需要上述命令中文件 "crd.yaml" 改为 "crd_v1beta1.yaml"。
```
Comment (qiancai) [inline zh/deploy-tidb-from-kubernetes-gke.md:103]:
```suggestion
> 对于 Kubernetes 1.16 之前的版本，Kubernetes 仅支持 v1beta1 版本的 CRD，你需要将上述命令中的 'crd.yaml' 修改为 'crd_v1beta1.yaml'。
```

File: en/deploy-tidb-from-kubernetes-gke.md [hunk 6]
```diff
@@ -97,10 +97,14 @@ If you see `Ready` for all nodes, congratulations! You've set up your first Kube
...
 
 ```shell
-kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/crd.yaml && \
+kubectl create -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/crd.yaml && \
 kubectl get crd tidbclusters.pingcap.com
 ```
 
+> **Note:**
+>
+> For Kubernetes < 1.16, it only support v1beta1 CRD, so we have to change the file "crd.yaml" in above commands to "crd_v1beta1.yaml".
```
Comment (qiancai) [inline en/deploy-tidb-from-kubernetes-gke.md:106]:
```suggestion
> For Kubernetes earlier than 1.16, only v1beta1 CRD is supported, so you need to change `crd.yaml` in the above command to `crd_v1beta1.yaml`.
```

File: zh/deploy-tidb-operator.md [hunk 7]
```diff
@@ -58,9 +58,13 @@ kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/
 
 ```shell
 wget https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/crd.yaml
-kubectl apply -f ./crd.yaml
+kubectl create -f ./crd.yaml
 ```
 
+> **注意：**
+> 
+> Kubernetes 1.16 之前版本，仅支持 v1beta1 版本的 CRD，需要上述命令中文件 "crd.yaml" 改为 "crd_v1beta1.yaml"。
```
Comment (qiancai) [inline zh/deploy-tidb-operator.md:66]:
```suggestion
> 对于 Kubernetes 1.16 之前的版本，Kubernetes 仅支持 v1beta1 版本的 CRD，你需要将上述命令中的 'crd.yaml' 修改为 'crd_v1beta1.yaml'。
```

File: en/deploy-tidb-operator.md [hunk 8]
```diff
@@ -58,9 +58,13 @@ If the server cannot access the Internet, you need to download the `crd.yaml` fi
 
 ```shell
 wget https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/crd.yaml
-kubectl apply -f ./crd.yaml
+kubectl create -f ./crd.yaml
 ```
 
+> **Note:**
+>
+> For Kubernetes < 1.16, it only support v1beta1 CRD, so we have to change the file "crd.yaml" in above commands to "crd_v1beta1.yaml".
```
Comment (qiancai) [inline en/deploy-tidb-operator.md:66]:
```suggestion
> For Kubernetes earlier than 1.16, only v1beta1 CRD is supported, so you need to change `crd.yaml` in the above command to `crd_v1beta1.yaml`.
```

File: en/get-started.md [hunk 9]
```diff
@@ -267,6 +267,10 @@ customresourcedefinition.apiextensions.k8s.io/tidbinitializers.pingcap.com creat
 customresourcedefinition.apiextensions.k8s.io/tidbclusterautoscalers.pingcap.com created
 ```
 
+> **Note:**
+>
+> For Kubernetes < 1.16, it only support v1beta1 CRD, so we have to change the file "crd.yaml" in above commands to "crd_v1beta1.yaml".
```
Comment (qiancai) [inline en/get-started.md:272]:
```suggestion
> For Kubernetes earlier than 1.16, only v1beta1 CRD is supported, so you need to change `crd.yaml` in the above command to `crd_v1beta1.yaml`.
```

File: zh/get-started.md [hunk 10]
```diff
@@ -262,6 +262,10 @@ customresourcedefinition.apiextensions.k8s.io/tidbinitializers.pingcap.com creat
 customresourcedefinition.apiextensions.k8s.io/tidbclusterautoscalers.pingcap.com created
 ```
 
+> **注意：**
+> 
+> Kubernetes 1.16 之前版本，仅支持 v1beta1 版本的 CRD，需要上述命令中文件 "crd.yaml" 改为 "crd_v1beta1.yaml"。
```
Comment (qiancai) [inline zh/get-started.md:267]:
```suggestion
> 对于 Kubernetes 1.16 之前的版本，Kubernetes 仅支持 v1beta1 版本的 CRD，你需要将上述命令中的 'crd.yaml' 修改为 'crd_v1beta1.yaml'。
```

File: zh/upgrade-tidb-operator.md (hunk 3 above)
Comment (KanShiori) [reply zh/upgrade-tidb-operator.md:34]: verified, it works for kubernetes v1.17 or less, but v1beta1 CRDs is incompatible with v1.18 or later

File: en/upgrade-tidb-operator.md [hunk 11]
```diff
@@ -17,12 +17,23 @@ This document describes how to upgrade TiDB Operator and Kubernetes.
...
+    * For Kubernetes versions >= 1.16:
 
-    ```shell
-    kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/v1.2.4/manifests/crd.yaml && \
-    kubectl get crd tidbclusters.pingcap.com
-    ```
+        {{< copyable "shell-regular" >}}
+
+        ```shell
+        kubectl replace -f https://raw.githubusercontent.com/pingcap/tidb-operator/v1.2.4/manifests/crd.yaml && \
```
Comment (DanielZhangQD) [inline en/upgrade-tidb-operator.md:25]: Should replace `v1.2.4` with `master`?

File: zh/upgrade-tidb-operator.md (hunk 2 above)
Comment (DanielZhangQD) [reply zh/upgrade-tidb-operator.md:25]: The `master` branch is for `dev` doc, which should match with the `master` branch in TiDB Operator.

Comment (KanShiori) [reply zh/upgrade-tidb-operator.md:25]: so, Should we need to change all versions to `master` in the dev doc?
It means that we should replace `master` to the release version when we release a new version.

Comment (DanielZhangQD) [reply zh/upgrade-tidb-operator.md:25]: Yes.

File: en/upgrade-tidb-operator.md (hunk 11 above)
Comment (KanShiori) [reply en/upgrade-tidb-operator.md]: Done

**Review by DanielZhangQD — APPROVED** [review]
LGTM

File: zh/deploy-tidb-from-kubernetes-gke.md [hunk 12]
```diff
@@ -94,10 +94,14 @@ kubectl get nodes
...
 
 ```shell
-kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/crd.yaml && \
+kubectl create -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/crd.yaml && \
 kubectl get crd tidbclusters.pingcap.com
 ```
 
+> **注意：**
+> 
+> 对于 Kubernetes 1.16 之前的版本，Kubernetes 仅支持 v1beta1 版本的 CRD，你需要将上述命令中的 'crd.yaml' 修改为 'crd_v1beta1.yaml'。
```
Comment (ran-huang) [inline zh/deploy-tidb-from-kubernetes-gke.md:103]:
```suggestion
> 对于 Kubernetes 1.16 之前的版本，Kubernetes 仅支持 v1beta1 版本的 CRD，你需要将上述命令中的 `crd.yaml` 修改为 `crd_v1beta1.yaml`。
```

File: zh/deploy-tidb-operator.md [hunk 13]
```diff
@@ -58,9 +58,13 @@ kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/
 
 ```shell
 wget https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/crd.yaml
-kubectl apply -f ./crd.yaml
+kubectl create -f ./crd.yaml
 ```
 
+> **注意：**
+> 
+> 对于 Kubernetes 1.16 之前的版本，Kubernetes 仅支持 v1beta1 版本的 CRD，你需要将上述命令中的 'crd.yaml' 修改为 'crd_v1beta1.yaml'。
```
Comment (ran-huang) [inline zh/deploy-tidb-operator.md:66]:
```suggestion
> 对于 Kubernetes 1.16 之前的版本，Kubernetes 仅支持 v1beta1 版本的 CRD，你需要将上述命令中的 `crd.yaml` 修改为 `crd_v1beta1.yaml`。
```

File: zh/get-started.md [hunk 14]
```diff
@@ -262,6 +262,10 @@ customresourcedefinition.apiextensions.k8s.io/tidbinitializers.pingcap.com creat
 customresourcedefinition.apiextensions.k8s.io/tidbclusterautoscalers.pingcap.com created
 ```
 
+> **注意：**
+> 
+> 对于 Kubernetes 1.16 之前的版本，Kubernetes 仅支持 v1beta1 版本的 CRD，你需要将上述命令中的 'crd.yaml' 修改为 'crd_v1beta1.yaml'。
```
Comment (ran-huang) [inline zh/get-started.md:267]:
```suggestion
> 对于 Kubernetes 1.16 之前的版本，Kubernetes 仅支持 v1beta1 版本的 CRD，你需要将上述命令中的 `crd.yaml` 修改为 `crd_v1beta1.yaml`。
```

**Review by ran-huang — APPROVED** [review]

**Pull request merged**